src/figuresClasses.ts

```diff
@@ -1,19 +1,70 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+type Color = 'red' | 'grin' | 'blur';
 
-}
+export class Triangle implements Figure {
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a + b <= c || c + a <= b || b + c <= a) {
+      throw new Error('the longest side of a triangle is'
+      + '>= than a sum of two others');
+    }
+  }
 
-export class Circle {
+  shape: string = 'triangle';
 
+  getArea(): number {
+    const semiPerimeter = (this.a + this.b + this.c) / 2;
+    const area = semiPerimeter * (semiPerimeter - this.a)
+    * (semiPerimeter - this.b) * (semiPerimeter - this.c);
+
+    return Math.floor(Math.sqrt(area) * 100) / 100;
```

Comment:
you round down to hundredths on each getArea, maybe better create a separate method for it?